### ISSUE TITLE
Android-device build failed. 1.8.1

### ISSUE THREAD

**Issue by Falkovic1993** [meteor/meteor#10582]
Hey Meteor team,

I'm trying to run my app on an android device it fails with the following message: 

```
   While running Cordova app for platform Android with options --buildConfig,C:\Work\eatie\.meteor\local\cordova-build\build.json,--device:
   Error: Command failed: C:\Work\eatie\.meteor\local\cordova-build\platforms\android\cordova\run --buildConfig C:\Work\eatie\.meteor\local\cordova-build\build.json --device --buildConfig
   C:\Work\eatie\.meteor\local\cordova-build\build.json --device
   C:\Work\eatie\.meteor\local\cordova-build\platforms\android\app\build\intermediates\manifests\full\debug\AndroidManifest.xml:47: AAPT: error: resource string/fb_app_id (aka io.eatie.app:string/fb_app_id) not
   found.

C:\Work\eatie\.meteor\local\cordova-build\platforms\android\app\build\intermediates\manifests\full\debug\AndroidManifest.xml:50: AAPT: error: resource string/fb_app_name (aka io.eatie.app:string/fb_app_name) not
   found.

   C:\Work\eatie\.meteor\local\cordova-build\platforms\android\app\build\intermediates\manifests\full\debug\AndroidManifest.xml:54: AAPT: error: resource string/fb_app_name (aka io.eatie.app:string/fb_app_name) not
   found.

   C:\Work\eatie\.meteor\local\cordova-build\platforms\android\app\build\intermediates\manifests\full\debug\AndroidManifest.xml:47: error: resource string/fb_app_id (aka io.eatie.app:string/fb_app_id) not found.
   C:\Work\eatie\.meteor\local\cordova-build\platforms\android\app\build\intermediates\manifests\full\debug\AndroidManifest.xml:50: error: resource string/fb_app_name (aka io.eatie.app:string/fb_app_name) not found.
   C:\Work\eatie\.meteor\local\cordova-build\platforms\android\app\build\intermediates\manifests\full\debug\AndroidManifest.xml:54: error: resource string/fb_app_name (aka io.eatie.app:string/fb_app_name) not found.
   error: failed processing manifest.
```
It's a while ago I've last time ran it, so not 100% sure it came with 1.8.1 or was exsisting before. 

In my ```mobile-config.js``` i have the following setup, which normally has worked fine.
```
App.configurePlugin('cordova-plugin-facebook4', {
	APP_ID: '************',
	APP_NAME: 'appName',
});
```
I have the following cordova plugins installed
```
cordova-plugin-facebook4@1.7.4
cordova-plugin-geolocation@4.0.1
cordova-plugin-inappbrowser@1.7.1
cordova-plugin-splashscreen@5.0.1
cordova-plugin-statusbar@2.4.1
```

And using ``` cordova": "^8.0.0", ```

Thanks for your time! :)

**Comment by Falkovic1993** [comment]
I got the build to work after removing the ```cordova-plugin-facebook4@1.7.4```, but still gives me some warnings that I don't remember seeing before. 

```
%% apple-ios version check failed ("C:\Work\eatie\.meteor\local\cordova-build\platforms\ios\cordova\apple_ios_version"), continuing anyways.
%% apple-ios version check failed ("C:\Work\eatie\.meteor\local\cordova-build\platforms\ios\cordova\apple_ios_version"), continuing anyways.
%% apple-xcode version check failed ("C:\Work\eatie\.meteor\local\cordova-build\platforms\ios\cordova\apple_xcode_version"), continuing anyways.
%% config file *-Info.plist requested for changes not found at C:\Work\eatie\.meteor\local\cordova-build\platforms\android\*-Info.plist, ignoring
%% config file *-Info.plist requested for changes not found at C:\Work\eatie\.meteor\local\cordova-build\platforms\android\*-Info.plist, ignoring
```

My guess is that somehow the new version of ```cordova-android``` don't get the information as it did before form the ```mobile-config.js``` file.

**Comment by Falkovic1993** [comment]
Figured out, it seems to be that the new version of ```cordova-android``` & the old version of  ```Cordova-plugin-facebook4``` didn't play well together, but after some search, I found this thread: https://github.com/jeduan/cordova-plugin-facebook4/issues/599#issuecomment-447555324

TLDR; Update your ```Cordova-plugin-facebook``` package to version 4 or higher. :)
